src/components/transmuters/EthTransmuterLooper.tsx

```diff
@@ -0,0 +1,652 @@
+  // anything that may need a value updated, and store those initial vals in dynamic state vars
+  const { data: transmuterLooperContractStaticState } = useReadContracts({
+    allowFailure: false,
+    contracts: [
+      {
+        // Get the underlying asset for the strategy.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "asset",
+        chainId: chain.id,
+      },
+      {
+        // Get the decimals used for the strategy and `asset`.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "decimals",
+        chainId: chain.id,
+      },
+      {
+        // Get the name the strategy is using for it's token
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "name",
+        chainId: chain.id,
+      },
+      {
+        // Get the performance fee charged on profits.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "performanceFee",
+        chainId: chain.id,
+      },
+      {
+        // Get the max the owner can deposit in asset.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "maxDeposit",
+        chainId: chain.id,
+        args: [address!],
+      },
+      {
+        // Get the max the owner can mint in shares.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "maxMint",
+        chainId: chain.id,
+        args: [address!],
+      },
+      {
+        // Get the max amount of shares the owner can redeem
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "maxRedeem",
+        chainId: chain.id,
+        args: [address!],
+      },
+      {
+        // Get the max amount of assets the owner can withdraw
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "maxWithdraw",
+        chainId: chain.id,
+        args: [address!],
+      },
+      {
+        // Gets the symbol the strategy is using for its tokens as a string. (yvSymbol)
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "symbol",
+        chainId: chain.id,
+      },
+    ],
+    query: {
+      select: ([
+        sharesTokenAddress,
+        decimals,
+        shareTokenName,
+        performanceFeeInBps,
+        maxDeposit,
+        maxMint,
+        maxRedeem,
+        maxWithdraw,
+        symbol,
+      ]) => {
+        return [
+          sharesTokenAddress,
+          Number(decimals),
+          shareTokenName,
+          (performanceFeeInBps as number) / 100,
+          BigInt(maxDeposit as number),
+          BigInt(maxMint as number),
+          BigInt(maxRedeem as number),
+          BigInt(maxWithdraw as number),
+          symbol,
+        ];
+      },
+    },
+  });
```

Comment:
when you got the `transmuterLooperContractStaticState`, right after the hook you can destruct the array.

```jsx
  const [
    asset,
    decimals,
    name,
    performanceFee,
    maxDeposit,
    maxMint,
    maxRedeem,
    maxWithdraw,
    symbol,
  ] = transmuterLooperContractStaticState ?? [];
```